js/activity.js

```diff
@@ -2,32 +2,43 @@ define(function (require) {
     var activity = require("sugar-web/activity/activity");
     var icon = require("sugar-web/graphics/icon");
     require("easel");
-    require("handlebars")
-    var shapes = require("activity/shapes")
+    require("handlebars");
+    var shapes = require("activity/shapes");
+
     // Manipulate the DOM only when it is ready.
     require(['domReady!'], function (doc) {
         // Initialize the activity.
         activity.setup();
+
         // Colorize the activity icon.
         var activityButton = document.getElementById("activity-button");
         activity.getXOColor(function (error, colors) {
             icon.colorize(activityButton, colors);
         });
+
+        // Handle New Button Click
         var newButton = document.getElementById("new-button");
         newButton.onclick = function () {
             new_positions();
         }
-        // Make the activity stop with the stop button.
+
+        // Handle Stop Button Click
```

Comment:
Comments like this are unnecessary, and not needed.
Please remove them.

Reply:
i have removed unnecessary comments